src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -118,12 +125,14 @@ public class HttpProtocolTests : HttpClientTest
         [InlineData("HTTP/1.1\t200 OK")]
         [InlineData("HTTP/1.1 200\tOK")]
         [InlineData("HTTP/1.1 200\t")]
-        public async Task GetAsync_InvalidStatusLine_ThrowsExceptionSkipForWinHttpHandlerCurlHandler(string responseString)
+        public async Task GetAsync_InvalidStatusLine_ThrowsExceptionOnManagedHandler(string responseString)
         {
-            if (PlatformDetection.IsUap || UseManagedHandler)
+            if (UseManagedHandler || PlatformDetection.IsFullFramework)
             {
+                // ManagedHandler and .NET Framework will throw HttpRequestException.
                 await GetAsyncThrowsExceptionHelper(responseString);
             }
+            // WinRT, WinHttpHandler, and CurlHandler will succeed.
```

Comment:
If all the other pass, what about moving the Assert.ThrowsAsync out of the helper - you could Assert.Throws in the callers.
Then for this case (WinRT/WinHttpHandler & CurlHandler) you just call - it will make sure no exception is thrown.

Anyway, it is not a high-pri as this is a negative test case, not supported by RFC.